src/app/pages/3_Multisequences_Alignment.py

```diff
@@ -0,0 +1,215 @@
+import configparser
+import os
+
+import streamlit as st
+import streamlit.components.v1 as components
+from common.render_method import render_markdown
+
+from app.common import setup
+from app.common.constants import MSA_CLEAN_ZIP, MSA_DF, MSA_VIEWER, NAVIGATE_WARNING_MD
+from app.common.setup import init_session_state_msa
+from app.frontend.download_data import download_msa_data
+from genetic_testing.sequence_tool.consensusSeq import ConsensusSeq, alignMs
+from genetic_testing.sequence_tool.msaViewer import MsaViewer, msa_cleaner
+
+setup.initialize()
+st.sidebar.image("src/app/Conservation X Labs CXL logo.png", use_column_width=True)
+
+# Initialize the Streamlit session state for this page
+init_session_state_msa()
+
+# Streamlit app header
+st.header("Sequence data alignment and cleanup")
+
+st.markdown(NAVIGATE_WARNING_MD)
+render_markdown("src/app/multisequence_alignment.md")
+
+st.markdown("### Align multisequences")
+
+# File uploader
+multifastaFile = st.file_uploader("Upload a FASTA file", type=["fasta"])
+
+with st.form("seq_viewer"):
+    # Button to trigger viewer
+    submitted = st.form_submit_button("Show sequence viewer")
+    if submitted:
+        # Make sure kill other viewer threads
+        if st.session_state[MSA_VIEWER] is None:
+            st.session_state[MSA_VIEWER] = MsaViewer(multifastaFile)
+            st.session_state[MSA_VIEWER].run()
+        else:
+            st.session_state[MSA_VIEWER].updateData(multifastaFile)
+
+        components.iframe(
+            "http://127.0.0.1:8050/", width=1200, height=1200, scrolling=True
```

Comment:
[nit] may be better to move this url to a constant shared by files. Not required